p2p/src/simulated/ingress.rs

```diff
@@ -1,32 +1,31 @@
 use super::{Error, Receiver, Sender};
 use crate::Channel;
 use commonware_cryptography::PublicKey;
-use commonware_utils::set::Ordered;
+use commonware_utils::set::{Ordered, OrderedAssociated};
 use futures::{
     channel::{mpsc, oneshot},
     SinkExt,
 };
 use rand_distr::Normal;
-use std::time::Duration;
+use std::{net::SocketAddr, time::Duration};
 
 pub enum Message<P: PublicKey> {
-    Update {
-        peer_set: u64,
-        peers: Ordered<P>,
-    },
     Register {
         channel: Channel,
         public_key: P,
         #[allow(clippy::type_complexity)]
         result: oneshot::Sender<Result<(Sender<P>, Receiver<P>), Error>>,
     },
+    Update {
```

Comment:
Arguably, we could refactor this to explicitly wait for "acks" from all functions. It seems like these control-related things are thought of as synchronous operations that we shouldn't pass over until resolved.

Reply:
Actually, we already wait on AddLink/RemoveLink/LimitBandwidth (the unique control ops) so this is sufficient.